eli5/sklearn/explain_prediction.py

```diff
@@ -111,3 +106,13 @@ def _add_intercept(X):
         return sp.hstack([X, intercept]).tocsr()
     else:
         return np.hstack([X, intercept])
+
+
+def _get_X(doc, vec=None, vectorized=False):
```

Comment:
heh, I extracted exactly the same utility (not commited) and named it almost the same (`_get_X_dense`)